proposals/nnnn-remote_atf_testing.md

```diff
@@ -0,0 +1,83 @@
+# Remote Automated testing
+
+* Proposal: [SDL-NNNN](NNNN-remote_atf_testing.md)
+* Author: [Alexander Kutsan](https://github.com/LuxoftAKutsan)
+* Status: **Awaiting review**
+* Impacted Platforms: [ATF]
+
+## Introduction
+
+This proposal is about modification of Automated Test Framework (ATF) for providing ability to test SDL on remote workstation usinig automated test scripts.   
+
+## Motivation
+
+Automated testing of SDL on the real OEM hardware would significantly reduce the amount of efforts at integration stage. 
+Also, it can ensure that during integration the business logic SDL will be ported successfully and will not be corrupted. 
+
+On various OEM platforms communication between HMI and SDL is performed through different transport, that may not be retrieved from the target (i.e. mqueue). So for ATF it's not possible to connect to SDL from HMI side and emulate HMI behavior.  
+This proposal suggests adding the optional proxy (relay) layer. This layer will connect to SDL, which is located on the OEM's target and expose API to ATF, which is located on other workstation via Transmission Control Protocol (TCP).
+
+## Proposed solution
+
+Create Remote ATF Adapter as separate application. This application will be executed on the same hardware with SDL and will provide API for communication with SDL and will control SDL life cycle.
```

Comment:
Change to: 
Create Remote ATF Adapter as a separate application. This application will be executed on the same hardware with SDL and will provide API for communication with SDL and will control the SDL life cycle.